src/index.ts

```diff
@@ -629,6 +629,26 @@ function nanoAddressDecoder(data: string): Buffer {
   return Buffer.from(decoded).slice(0, -5);
 }
 
+// Remove staring zeros from buffer
+function flowDecode(data: string): Buffer {
+  return Buffer.from(rskStripHexPrefix(data).replace(/^0+/, ''), 'hex');
+}
+
+// https://github.com/onflow/flow-go/blob/master/model/flow/address.go#L51
+// If b is larger than 8, b will be cropped from the left.
+// If b is smaller than 8, b will be appended by zeroes at the front.
+function flowEncode(data: Buffer): string {
+  const AddressLength = 8;
+  let addrBytes = Buffer.alloc(AddressLength, 0x00);
+
+  if (data.length > 8) {
+    addrBytes = data.slice(-8);
```

Comment:
Maybe replace the magic numbers with the constant defined above:

```suggestion
  if (data.length > AddressLength) {
    addrBytes = data.slice(-AddressLength);
```